modules/API/API.js

```diff
@@ -271,7 +271,9 @@ function initCommands() {
                 return;
             }
 
-            conference.startRecording(recordingConfig);
+            conference.startRecording(recordingConfig).catch(() => {
```

Comment:
Is it expected to regularly have an unhandled rejected promise at this level. I'm less familiar with the client work but it seems like this may not be an expected state and worth logging or trying to eliminate it?

Reply:
So the function returns Promise. That would basically mean that rejected promise is always expected :)

I would agree with you about the logging but in our case we anyway have pretty extensive logging in lib-jitsi-meet, so I don't think there's anything that we can log here that will bring any value.

Reply:
Does the API provide any feedback on whether or not the action succeeded?

Reply:
Things like `logger.error('Conference is not defined');` are logged above, yet you don't want to log an error which would give a clue on why things have failed? Is there any indication to the API user that things went wrong when the promise is rejected?